nsxt/resource_nsxt_policy_ip_block.go

```diff
@@ -165,6 +172,7 @@ func resourceNsxtPolicyIPBlockCreate(d *schema.ResourceData, m interface{}) erro
 		obj.CidrList = cidrList
```

Comment:
I think there might be a problem here - are all those attributes dependent on `len(cidrList) > 0`
Is it possible to specify `range_list` or `is_subnet_exclusive` without specifying `cidr_list`?